corehq/apps/geospatial/routing_solvers/pulp.py

```diff
@@ -3,10 +3,36 @@
 import pulp
 import copy
 
+from dataclasses import dataclass
 from .mapbox_utils import validate_routing_request
 from corehq.apps.geospatial.routing_solvers.base import DisbursementAlgorithmSolverInterface
 
 
+@dataclass
+class Parameters:
+    min_cases_per_user: int
+    max_cases_per_user: int
+    max_case_distance: int
+    max_case_travel_time: int
+    user_count: int
+    case_count: int
+
+    def __init__(self, config, user_count, case_count):
+        self.user_count = user_count
+        self.case_count = case_count
+
+        default_max_cases = int(case_count / user_count) + 1 if user_count else 0
+        self.max_cases_per_user = config.max_cases_per_user or default_max_cases
+
+        self.min_cases_per_user = config.min_cases_per_user or 0
+        self.max_case_distance = config.max_case_distance
+        self.max_case_travel_time = config.max_case_travel_time if config.supports_travel_mode else None
+
+    @property
+    def max_travel_time_seconds(self):
```

Comment:
nit: just curious is this function belongs in a dataclass. Is it here like a calculated property? Could this be in __init__ itself or is it calculated by max_case_travel_time can be updated?

Reply:
Good point. Since we're using only the seconds I'll calculate it and store the calculated version in memory